FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Posix/FreeRTOSConfig.h

```diff
@@ -43,10 +43,10 @@
 #define configUSE_TICK_HOOK						1
 #define configUSE_DAEMON_TASK_STARTUP_HOOK		1
 #define configTICK_RATE_HZ						( 1000 ) /* In this non-real time simulated environment the tick frequency has to be at least a multiple of the Win32 tick frequency, and therefore very slow. */
-#define configMINIMAL_STACK_SIZE				( ( unsigned short ) 70 ) /* In this simulated case, the stack only has to hold one small structure as the real stack is part of the win32 thread. */
-#define configTOTAL_HEAP_SIZE					( ( size_t ) ( 65 * 1024 ) )
+#define configMINIMAL_STACK_SIZE				( ( unsigned short ) 100000 ) /* In this simulated case, the stack only has to hold one small structure as the real stack is part of the win32 thread. */
```

Comment:
May want to update the comment here.

Reply:
well actually .. why is the MINIMAL_STACK_SIZE that big? id set it to maybe around 2000 ... 100000 is massive.

Reply:
Oh thanks for catching that, it just needed to be the PTHREAD_STACK_MIN, I accidentally over-wrote that when doing the commit.